assets/charts/control-plane/kubelet/container-image/README.md

```diff
@@ -0,0 +1,57 @@
+# Lokomotive Kubelet Image
```

Comment:
Do we have some docs about updating Kubernetes version? It would be nice if we could like to this file from there. I think `assets/charts/control-plane/kubelet/container-image/README.md` will be difficult to find. Also, this image is not only used by the chart but also in Ignition files, so honestly I'm not sure if this is the right place for it...

Reply:
We don't have a document for updating k8s. The instructions are close to `Dockerfile`.

Reply:
Can we fork github.com/poseidon/kubelet and put this there then?

Reply:
Isn't this essentially the same?

Reply:
I think it's easier to find by mapping `quay.io/kinvolk/kubelet` -> `github.com/kinvolk/kubelet` than to `github.com/kinvolk/lokomotive/assets/charts/control-plane/kubelet/container-image`.

Reply:
Also, having forked repo would resolve #221.